transactions/transactor.go

```diff
@@ -68,25 +68,33 @@ func (t *Transactor) SetRPC(rpcClient *rpc.Client, timeout time.Duration) {
 }
 
 // SendTransaction is an implementation of eth_sendTransaction. It queues the tx to the sign queue.
-func (t *Transactor) SendTransaction(ctx context.Context, args SendTxArgs) (gethcommon.Hash, error) {
+func (t *Transactor) SendTransaction(ctx context.Context, sendArgs SendTxArgs, password string, signArgs *sign.TxArgs, verify sign.VerifyFunc) sign.Result {
 	if ctx == nil {
 		ctx = context.Background()
 	}
 
 	completeFunc := func(acc *account.SelectedExtKey, password string, signArgs *sign.TxArgs) (sign.Response, error) {
```

Comment:
I think we need to get rid of this function at all and just get the current account here or pass it to this function

Reply:
What replaces backend.SendTransaction then? How do we call transactor.SendTransaction directly from the RPC interface then?

Reply:
In backend.SendTransaction you can get current active account and pass it into this function.